docs/api/routers/RoutersAPI.md

```diff
@@ -51,7 +51,7 @@ Typically this should return a navigation state, with the following form:
 
 If the router has handled the action externally, or wants to swallow it without changing the navigation state, this function will return `null`.
 
-### `getComponentRouteName(routeName)`
+### `getComponentForRouteName(routeName)`
```

Comment:
I'm not familiar enough with the routers to have input on the other corrections in this PR, but changing `getComponentRouteName` to `getComponentForRouteName` is a good catch.